src/js/binaryen.js-post.js

```diff
@@ -524,69 +524,63 @@ function wrapModule(module, self) {
   // their own Literals, as the C API handles them by value, which means
   // we would leak them. Instead, Const creation is fused together with
   // an intermediate stack allocation of this size to pass the value.
-  var sizeOfLiteral = _BinaryenSizeofLiteral();
+  const sizeOfLiteral = _BinaryenSizeofLiteral();
 
   // 'Expression' creation
   self['block'] = function(name, children, type) {
-    return preserveStack(function() {
-      return Module['_BinaryenBlock'](module, name ? strToStack(name) : 0,
-                                      i32sToStack(children), children.length,
-                                      typeof type !== 'undefined' ? type : Module['none']);
-    });
+    return preserveStack(() => (
+      Module['_BinaryenBlock'](module, name ? strToStack(name) : 0,
+                               i32sToStack(children), children.length,
+                               typeof type !== 'undefined' ? type : Module['none'])
+    ));
```

Comment:
The additional parenthesis here and at other locations look somewhat odd to me:

```suggestion
    return preserveStack(() =>
      Module['_BinaryenBlock'](module, name ? strToStack(name) : 0,
                               i32sToStack(children), children.length,
                               typeof type !== 'undefined' ? type : Module['none'])
    );
```

Or is there a good reason to do it the other way?

Reply:
Removed this